ciao-cli/image.go

```diff
@@ -97,18 +87,11 @@ var imageCommand = &command{
 }
 
 type imageAddCommand struct {
-	Flag            flag.FlagSet
-	name            string
-	id              string
-	containerFormat imageContainerFormat
-	diskFormat      imageDiskFormat
-	minDiskSize     int
```

Comment:
it's not only disk size, what about memory, you need to specifiy which is the minimal memory a OS image needs for running. Then, workloads can start from that minimal required ram.